packages/strategy-tests/src/lib.rs

```diff
@@ -578,9 +579,8 @@ impl Strategy {
                                     .unwrap_or(&0)
                                     < &24u64
                             })
-                            .take(10)
                             .cloned()
```

Comment:
You don't need to clone all filtered identities. Only those 10 what you randomly selected.

Reply:
Fixed